docs/application/native/guides/ui/tizen-ws-shell.md

```diff
@@ -0,0 +1,151 @@
+# Tizen Window System Shell
+
+
+The Tizen window system shell (TZSH) is an interface for manipulating windows of the sytem graphical user interfaces (GUI) services such as the quickpanel and the softkey. It provides C libraries that allow developers to control windows of system GUI services and be notified state changes of them within an application.
+
+Since the system GUI services are different processes from the application, all of operations of the TZSH are performed using the inter-process communication (IPC) through the display server. Below figure shows a simple S/W architecture of the TZSH.
```

Comment:
Since, the system GUI services execute different processes in the application, all of operations of the TZSH are performed using the inter-process communication (IPC) through the display server. The following figure illustrates a simple software architecture of the TZSH:

Reply:
"system GUI services execute different processes in the application"

It looks like system GUI services is launched by the user application.
If so, it is not what I mean. I wanted to explain that each system GUI services are running in separate process. Thus IPC is needed to work between system GUI service processes and separated user application process. So I just use below sentence. Or could you let me know better sentence what I mean?

"the system GUI services are different processes from the application"

Reply:
Since, the system GUI service runs different processes, all of operations of the TZSH are performed using the inter-process communication (IPC). It is needed to enable  communication between system GUI service processes and separated user application process using the display server. The following figure illustrates a simple software architecture of the TZSH:

Reply:
Fine.